.github/workflows/validate.yml

```diff
@@ -86,3 +86,17 @@ jobs:
       # We validate all commits as we only fetched the number of commits in the PR above,
       # by default git-validation has some special github action handling but that seems broken.
       run: make .install.gitvalidation && git-validation -no-github
+
+  go-tidy:
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v5
+    - uses: actions/setup-go@v5
+      with:
+        go-version: 1.25.x
+        # See comment on lint task
+        cache-dependency-path: "**/go.sum"
+    - name: tidy
+      run: make tidy
+    - name: check if tree is clean
+      run: hack/tree_status.sh
```

Comment:
This file says `-run \"make vendor\" and commit all changes` , that target was renamed.